.travis.yml

```diff
@@ -1,5 +1,3 @@
-sudo: true
```

Comment:
Do we still need travis?

Reply:
As I understand it, this is temporary for comparison.

Reply:
We will remove .travis.yml in next PR.